extensions/objects/protractor.js

```diff
@@ -205,6 +205,41 @@ var SanitizedUrlEditor = function(elem) {
   };
 };
 
+var SetOfNormalizedStringEditor = function(elem) {
+  const _getLength = async function() {
+    var items = (
+      await elem.all(by.repeater('item in localValue track by $index')));
+    return items.length;
+  };
+
+  const addItem = async function(value) {
+    var listLength = await _getLength();
+    await elem.element(by.css('.protractor-test-add-list-entry')).click();
+    return await NormalizedStringEditor(
+      elem.element(
+        await by.repeater(
+          'item in localValue track by $index').row(listLength))
```

Comment:
Can we use a CSS selector instead like we do in other files?

Reply:
Also, we might not need an `await` here because `by.repeater` does not return a promise.

Reply:
I'm not sure how to do this - the NormalizedStringEditor is actually just a schema-based-list-editor of unicode. I've looked at the html + css but I don't see any protractor classes.

Is there a file I can look at for an example?

Reply:
Oh, you can create a new class of the pattern "protractor-test-.." (as used in other places) and then use that as a selected in the e2e test.